doc/references.bib

```diff
@@ -836,3 +836,25 @@ @article{schuld2019evaluating
   year={2019},
   publisher={APS}
 }
+
+@article{arrazola2018using,
+  title={Using Gaussian Boson Sampling to Find Dense Subgraphs},
+  author={Arrazola, Juan Miguel and Bromley, Thomas R},
+  journal={Physical Review Letters},
+  volume={121},
+  number={3},
+  pages={030503},
+  year={2018},
+  publisher={APS}
+}
+
```

Comment:
Do we mention Kamil's paper on embedding graphs into GBS anywhere in the documentation? We should

Reply:
Fixed!